### PR TITLE
updated KB registry.

### PR DESCRIPTION
**Pull request by mjshastha**
updated KB registries from docker to aqua registry.

Additionally, updated starboard image to latest.

### CODE REVIEW

File: enforcers/kube_enforcer/kubernetes_and_openshift/manifests/kube_enforcer_advanced/003_kube_enforcer_deploy.yaml [hunk 1]
```diff
@@ -78,7 +78,7 @@ spec:
             - name: CLUSTER_NAME
               value: "Default-cluster-name"   # Cluster display name in aqua enterprise.
             - name: AQUA_KB_IMAGE_NAME
-              value: "aquasec/kube-bench:v0.7.3"
+              value: "registry.aquasec.com/kube-benchv0.7.3:"
```
Comment (deven0t) [inline enforcers/kube_enforcer/kubernetes_and_openshift/manifests/kube_enforcer_advanced/003_kube_enforcer_deploy.yaml:81]: colon(:) is misplaced

**Review by deven0t — APPROVED** [review]

**Comment by mjshastha** [comment]
@semyonmor Can you merge this pr?

**Review by yariv-freifeld-aqua — APPROVED** [review]

**Review by seanposner — APPROVED** [review]

**Review by seanposner — CHANGES_REQUESTED** [review]
PR is failing

**Comment by mjshastha** [comment]
@seanposner Can you merge the changes?

**Comment by mjshastha** [comment]
@seanposner Can you merge the changes?

**Review by yariv-freifeld-aqua — APPROVED** [review]

**Review by yariv-freifeld-aqua — APPROVED** [review]

**Pull request merged**